tests/console/btrfs_check.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: btrfs_check
+# Summary: Test that btrfs is setup properly
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    my $kiwi_volumes = qq(
+/
+/.snapshots
+/boot/grub2/i386-pc
+/boot/grub2/x86_64-efi
+/home
+/opt
+/root
+/srv
+/tmp
+/usr/local
+/var);
+
+    my $test_volumes = script_output("lsblk --list --noheadings --output MOUNTPOINTS /dev/vda3 | sort");
```

Comment:
I would rather see a list of subvolumes, hence using btrfs commands, but that might be a matter of taste. The good part is that with subvolumes you would use `/` rather than a drive. Drives might have different labels, depends on hypervisor mostly

Reply:
Nice, I'll see if I can remove all the "unnecessary" info and keep only the last column for comparison purposes.